tests/crm/test_crm.py

```diff
@@ -0,0 +1,881 @@
+    original_crm_stats_acl_counter_available = std_out
+
+    apply_acl_config(duthost, "test_acl_counter", collector)
+
+    # Get new "crm_stats_acl_counter" used and available counter value
+    get_acl_counter_stats = "redis-cli --raw -n 2 HMGET {acl_tbl_key} crm_stats_acl_counter_used \
+        crm_stats_acl_counter_available".format(acl_tbl_key=acl_tbl_key)
+    new_crm_stats_acl_counter_used, new_crm_stats_acl_counter_available = get_crm_stats(get_acl_counter_stats, duthost)
+
+    # Verify "crm_stats_acl_counter_used" counter was incremented
+    pytest_assert(new_crm_stats_acl_counter_used - crm_stats_acl_counter_used == 2, \
+        "\"crm_stats_acl_counter_used\" counter was not incremented")
+
+    used_percent = get_used_percent(new_crm_stats_acl_counter_used, new_crm_stats_acl_counter_available)
+    if used_percent < 1:
+        # Preconfiguration needed for used percentage verification
+        nexthop_group_num = get_entries_num(new_crm_stats_acl_counter_used, new_crm_stats_acl_counter_available)
+
+        apply_acl_config(duthost, "test_acl_counter", collector, nexthop_group_num)
+
+        logger.info("Waiting {} seconds for SONiC to update resources...".format(SONIC_RES_UPDATE_TIME))
+        # Make sure SONIC configure expected entries
+        time.sleep(SONIC_RES_UPDATE_TIME)
+
+        new_crm_stats_acl_counter_used, new_crm_stats_acl_counter_available = get_crm_stats(get_acl_counter_stats, duthost)
+
+    crm_stats_acl_counter_available = new_crm_stats_acl_counter_available + new_crm_stats_acl_counter_used
+
+    # Verify thresholds for "ACL entry" CRM resource
+    verify_thresholds(duthost, crm_cli_res=RESTORE_CMDS["crm_cli_res"], crm_used=new_crm_stats_acl_counter_used,
+        crm_avail=new_crm_stats_acl_counter_available)
+
+    # Remove ACL
+    duthost.command("acl-loader delete")
+
+    # Make sure CRM counters updated
+    time.sleep(CRM_UPDATE_TIME)
+
+    # Get new "crm_stats_acl_counter" used and available counter value
+    new_crm_stats_acl_counter_used, new_crm_stats_acl_counter_available = get_crm_stats(get_acl_counter_stats, duthost)
+
+    # Verify "crm_stats_acl_counter_used" counter was decremented
+    pytest_assert(new_crm_stats_acl_counter_used - crm_stats_acl_counter_used == 0, \
+        "\"crm_stats_acl_counter_used\" counter was not decremented")
+
+    # Verify "crm_stats_acl_counter_available" counter was incremented
+    pytest_assert(new_crm_stats_acl_counter_available - crm_stats_acl_counter_available >= 0, \
+        "\"crm_stats_acl_counter_available\" counter was not incremented")
+
+    # Verify "crm_stats_acl_counter_available" counter was equal to original value
+    pytest_assert(original_crm_stats_acl_counter_available - new_crm_stats_acl_counter_available == 0, \
+        "\"crm_stats_acl_counter_available\" counter is not equal to original value")
+
+
+def test_crm_fdb_entry(duthost, testbed):
+    if testbed["topo"]["name"] != "t0":
```

Comment:
Why only ```t0```, probably it should be supported on ```t0``` variations.

Reply:
Fixed.